app/api/schema/settings.py

```diff
@@ -200,10 +200,15 @@ class Meta:
     smtp_port = fields.Integer(allow_none=True)
     smtp_encryption = fields.Str(allow_none=True)  # Can be tls, ssl, none
 
+    # Event Invoices settings
+    invoices_sent_day = fields.Integer(allow_none=False, default=1)
+    invoices_timezone = fields.Str(allow_none=False, default="UTC")
```

Comment:
Why is this needed?

Reply:
TO specify on what day the event invoice should be sent every month.

Reply:
timezone

Reply:
Suppose we are sending it on the first of every month, it is used to specify in which timezone  it should be sent.(1st January in IST or PDT)

Reply:
https://github.com/fossasia/open-event-frontend/issues/3126#issue-455093188

Reply:
invoices_sent_day -> invoice_sending_{day, timezone}

Reply:
@iamareebjamal Done.